components/slider/__tests__/__snapshots__/demo.test.js.snap

```diff
@@ -955,7 +955,7 @@ exports[`renders ./components/slider/demo/show-tooltip.md correctly 1`] = `
   />
   <div>
     <div
-      class="ant-tooltip ant-slider-tooltip"
+      class="ant-tooltip"
```

Comment:
Miss `ant-slider-tooltip`

Reply:
will be resolved after updating snapshots

Reply:
strange, not resolved 🤔